spec/serializers/my_request/requests/show_serializer_spec.rb

```diff
@@ -16,32 +17,35 @@
   subject { JSON.parse(serialization.to_json) }
 
   it 'wraps content in key reflecting model name' do
-    expect(subject.keys).to match ["request"]
+    expect(subject.keys).to match ['request']
   end
 
   it 'contains only id, title, description, reward, status and offers' do
     expected_keys = %w[id title description reward status offers]
-    expect(subject["request"].keys).to match expected_keys
+    expect(subject['request'].keys).to match expected_keys
   end
 
   it 'offer contains only id, email, message and status' do
-    expected_keys = %w[id email message status]
-    expect(subject["request"]["offers"].first.keys).to match expected_keys
+    expected_keys = %w[id email status]
+    expect(subject['request']['offers'].first.keys).to match expected_keys
+  end
+
+  it 'conversation has a specific structure' do
```

Comment:
This will be next story?

Reply:
Yes, added, then realized it was out of scope, and forgot to remote fully.